components/CountryFormPage/index.js

```diff
@@ -21,12 +24,27 @@ export default function CountryForm({ submitNewCountry }) {
           <StyledDiv>
             <StyledLabel htmlFor="name">Land hinzufügen: </StyledLabel>
             <StyledInput
+              maxLength="60"
               name="name"
               id="name"
               type="text"
               placeholder="Österreich, Australien..."
               required
             />
+            <StyledLabel htmlFor="startdate">Startdatum:</StyledLabel>
+            <StyledInput
+              maxLength="60"
```

Comment:
`maxlength` probably doesn't do anything since it only applies to input types like `text`, `email`, etc.
You _can_ add a `min` and `max` attribute, though.